command/bridge/withdraw/erc1155/withdraw_erc1155.go

```diff
@@ -0,0 +1,261 @@
+
+	for i, receiverRaw := range wp.Receivers {
+		receivers[i] = ethgo.Address(types.StringToAddress(receiverRaw))
+		amountRaw := wp.Amounts[i]
+		tokenIDRaw := wp.TokenIDs[i]
+
+		amount, err := types.ParseUint256orHex(&amountRaw)
+		if err != nil {
+			outputter.SetError(fmt.Errorf("failed to decode provided amount %s: %w", amountRaw, err))
+
+			return
+		}
+
+		tokenID, err := types.ParseUint256orHex(&tokenIDRaw)
+		if err != nil {
+			outputter.SetError(fmt.Errorf("failed to decode provided token id %s: %w", amountRaw, err))
+
+			return
+		}
+
+		amounts[i] = amount
+		TokenIDs[i] = tokenID
+	}
+
+	// withdraw tokens transaction
+	txn, err := createWithdrawTxn(receivers, amounts, TokenIDs)
+	if err != nil {
+		outputter.SetError(fmt.Errorf("failed to create tx input: %w", err))
+
+		return
+	}
+
+	receipt, err := txRelayer.SendTransaction(txn, senderAccount)
+	if err != nil {
+		outputter.SetError(fmt.Errorf("failed to send withdrawal transaction (receivers: %s, amounts: %s, token ids: %s): %w",
+			strings.Join(wp.Receivers, ", "), strings.Join(wp.Amounts, ", "), strings.Join(wp.TokenIDs, ", "), err))
+
+		return
+	}
+
+	if receipt.Status == uint64(types.ReceiptFailed) {
+		outputter.SetError(fmt.Errorf("failed to execute withdrawal transaction (receivers: %s, amounts: %s, token ids: %s)",
+			strings.Join(wp.Receivers, ", "), strings.Join(wp.Amounts, ", "), strings.Join(wp.TokenIDs, ", ")))
+
+		return
+	}
+
+	exitEventID, err := extractExitEventID(receipt)
+	if err != nil {
+		outputter.SetError(fmt.Errorf("failed to extract exit event: %w", err))
+
+		return
+	}
+
+	outputter.SetCommandResult(
+		&withdrawResult{
+			Sender:      senderAccount.Address().String(),
+			Receivers:   wp.Receivers,
+			Amounts:     wp.Amounts,
+			TokenIDs:    wp.TokenIDs,
+			ExitEventID: strconv.FormatUint(exitEventID.Uint64(), 10),
+			BlockNumber: strconv.FormatUint(receipt.BlockNumber, 10),
+		})
+}
+
+// createWithdrawTxn encodes parameters for withdraw function on child chain predicate contract
+func createWithdrawTxn(receivers []ethgo.Address, amounts, TokenIDs []*big.Int) (*ethgo.Transaction, error) {
+	withdrawFn := &contractsapi.WithdrawBatchChildERC1155PredicateFn{
+		ChildToken: types.StringToAddress(wp.TokenAddr),
+		Receivers:  receivers,
+		Amounts:    amounts,
+		TokenIDs:   TokenIDs,
+	}
+
+	input, err := withdrawFn.EncodeAbi()
+	if err != nil {
+		return nil, fmt.Errorf("failed to encode provided parameters: %w", err)
+	}
+
+	addr := ethgo.Address(types.StringToAddress(wp.PredicateAddr))
+
+	return &ethgo.Transaction{
+		To:    &addr,
+		Input: input,
+	}, nil
+}
+
+// extractExitEventID tries to extract exit event id from provided receipt
+func extractExitEventID(receipt *ethgo.Receipt) (*big.Int, error) {
+	var exitEvent contractsapi.L2StateSyncedEvent
+	for _, log := range receipt.Logs {
+		doesMatch, err := exitEvent.ParseLog(log)
+		if !doesMatch {
+			continue
+		}
+
+		if err != nil {
```

Comment:
Maybe handle error first (if error occurs I suppose `doesMatch` will be false in most cases) and then match case later:
```
		doesMatch, err := exitEvent.ParseLog(log)		
		if err != nil {
			return nil, err
		}

                if doesMatch {
			return exitEvent.ID, nil
		}
```

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1332/commits/8bb5ab6d1c470d7037b91d446da3857e673268b5